Makefile

```diff
@@ -1,4 +1,34 @@
-.PHONY: fmt-check lint fmt
+prefix = /usr
+
+all:
+	: # do nothing
+
+build:
+
+	cd mgr && shards install && shards build
+
+install: build
```

Comment:
Instead of install step here can we move makefile to `debian/rules` file? Ex: https://github.com/gluster/glusterfs-debian/blob/bionic-glusterfs-10/debian/rules